dandi/dandiarchive.py

```diff
@@ -253,21 +187,11 @@ class _dandi_url_parser:
     ]
     # We might need to remap some assert_types
     map_asset_types = {"dandiset": "folder"}
-    # And lets create our mapping into girder instances from known_instances:
-    map_to = {"girder": {}, "api": {}}
-    for (
-        metadata_version,
-        girder,  # noqa: F402
-        gui,
-        redirector,
-        api,
-    ) in known_instances.values():
+    map_to = {}
+    for (gui, redirector, api) in known_instances.values():
         for h in (gui, redirector):
-            if h:
-                if girder:
-                    map_to["girder"][h] = girder
-                if api:
-                    map_to["api"][h] = api
+            if h and api:
+                map_to[h] = api
 
     @classmethod
     def parse(cls, url, *, map_instance=True):
```

Comment:
below, Example urls with `folder/longid` and `collection/` - all girder based, are gone now, could be removed